env/env.go

```diff
@@ -14,4 +14,5 @@ const (
 	InjectorMountSysrqTrigger = "CHAOS_INJECTOR_MOUNT_SYSRQ_TRIGGER"
 	InjectorTargetPodHostIP   = "TARGET_POD_HOST_IP"
 	InjectorChaosPodIP        = "CHAOS_POD_IP"
+	Hostname                  = "HOSTNAME"
```

Comment:
Is it a default environment variables for pods? I could not find any information about that in the Kubernetes documentation. Otherwise, or if we are not sure, we can still create the chaos pods with an environment variable like:
```yaml
      env:
        - name: MY_POD_NAME
          valueFrom:
            fieldRef:
              fieldPath: metadata.name
```
[as suggested in the Kubernetes documentation about environment variables](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/#use-pod-fields-as-values-for-environment-variables).

Reply:
oh, hmm, good point, i will change